cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -340,6 +340,7 @@ features:
       suggestionMessage2: Page 1
     Title: CDAP | Control Center
   DataPrepServiceControl:
+    btnLabel: Enable Data Preparation service
```

Comment:
since this is a button, shouldn't it be `Enable data preparation service`?

Reply:
Enable Data Preparation. User doesn't need to know its a service, right?

Reply:
`Data Preparation` is a proper noun in CDAP, so it should be capitalized